client/me/purchases/credit-cards/credit-card-delete.jsx

```diff
@@ -17,15 +18,17 @@ const CreditCardDelete = React.createClass( {
 		this.props
 			.deleteStoredCard( this.props.card )
 			.then( () => {
-				this.props.successNotice( this.translate( 'Card deleted successfully' ) );
+				this.props.successNotice( this.props.translate( 'Card deleted successfully' ) );
 			} )
 			.catch( error => {
 				this.props.errorNotice( error.message );
 			} );
 	},
 
 	renderDeleteButton: function() {
-		const text = this.props.isDeleting ? this.translate( 'Deleting ' ) : this.translate( 'Delete' );
+		const text = this.props.isDeleting
+			? this.props.translate( 'Deleting ' )
```

Comment:
I updated to `Deleting` because Eslint complains about using three dots instead of ellipsis character.